### DISCUSSION TITLE
Is the Kallichore source going to be released under the same licence?

### DISCUSSION THREAD

**Discussion by TimTaylor** [posit-dev/positron#5865] [Q&A]
As per title... will the source code for Kallichore be released under the same licence as Positron?

**Comment by jmcphers** [comment]
At present there are no plans to release Kallichore's source code under the same Elastic license as Positron.

**Comment by TimTaylor** [comment]
Sorry I was possibly at a bit too specific in the question ... Are there any plans to release the Kallichore source code (under any licence) to allow users to build directly from source? Or is the intention to only supply it as a binary?

**Reply by jmcphers** [reply]
Just as a binary. We'll provide it for more platforms at release than we do at present (e.g. an arm64 build is in the works)

**Comment by jmcphers** [comment]
Update: We've decided to release Kallichore under the same licensing terms as Positron itself, Elastic 2.0. You can find the repository here: https://github.com/posit-dev/kallichore
